progenitor-client/src/progenitor_client.rs

```diff
@@ -6,7 +6,7 @@
 
 use std::ops::{Deref, DerefMut};
 
-use bytes::Bytes;
+pub use bytes::Bytes;
```

Comment:
does this need to be pub? `Error::InvalidResponsePayload`'s inner type isn't `pub`